test/karma.config.js

```diff
@@ -1,4 +1,7 @@
-// eslint-disable-next-line filenames/match-regex, import/no-commonjs
+// eslint-disable-next-line filenames/match-regex, import/no-commonjs, @typescript-eslint/no-var-requires
+process.env.CHROME_BIN = require('puppeteer').executablePath()
```

Comment:
If we installed `playwright` instead, it'll also install firefox, chrome and webkit which means we could test in all 3!

```suggestion
process.env.CHROME_BIN = require('playwright').chromium.executablePath()
process.env.FIREFOX_BIN = require('playwright').firefox.executablePath();
process.env.WEBKIT_HEADLESS_BIN = require('playwright').webkit.executablePath();
```

We'd then switch `browsers: ['ChromeHeadless'],` to `browsers: ['ChromeHeadless', 'FirefoxHeadless', 'WebkitHeadless'],`